tracee/tracee.go

```diff
@@ -30,6 +30,7 @@ type TraceeConfig struct {
 	PerfBufferSize        int
 	BlobPerfBufferSize    int
 	OutputPath            string
+	GoTemplatePath        map[string]string
```

Comment:
where is this used?

Reply:
This came from the previous check-in. I've removed it now.